lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -79,7 +78,7 @@ def main():
 
         # search the pedestal calibration file
 
-        pedestal_file = f"{output_dir}/drs4_pedestal.Run{ped_run}.0000.fits"
+        pedestal_file = f"{output_dir}/drs4_pedestal.Run{'%05d'%ped_run}.0000.fits"
```

Comment:
Please don't mix % formatting with f-strings, just use `f"{output_dir}/drs4_pedestal.Run{ped_run:05d}.0000.fits"`

Reply:
Like everywhere else...

Reply:
Hi @maxnoe, sorry, I do not understand the comment what do you mean?

Reply:
Please replace
```
pedestal_file = f"{output_dir}/drs4_pedestal.Run{'%05d'%ped_run}.0000.fits"
```

with 

```
pedestal_file = f"{output_dir}/drs4_pedestal.Run{ped_run:05d}.0000.fits"
```

Like you used everywhere else in this file.

Reply:
> shouldn't all these be integers?

yes, they must be integer

Reply:
The current code is using the old `%` based formatting to create the string `0XXXX` and then uses an f-string to insert that into the file name. This doesn't make sense. Just use the f-string.

Reply:
Ah, yes, that is an overlook (I made test), I am going to change it